### DISCUSSION TITLE
How to supress email sending when creating account via internal_request

### DISCUSSION THREAD

**Discussion by whysthatso** [janko/rodauth-rails#291] [Q&A] [answered]
Is this possible?

I'm imagining something like this:

```
RodauthApp.rodauth.create_account(login: "user@example.com", password: "secret123", verification_email: false)
```

Or is this something that should be configured when enabling the features?

**Comment by janko** [comment, marked as answer]
If you just want to prevent emails being sent, and still keep the account unverified on creation, you can override `send_verify_account_email`:

```rb
send_verify_account_email do
  return if internal_request? 
  # implementation
end
```
